### ISSUE TITLE
Request: Option to use current folder for saving sent message

### ISSUE THREAD

**Issue by emtiu** [roundcube/roundcubemail#7166]
I'm used to a certain behaviour from (neo)mutt that would make my life easier in roundcube as well: When composing a new message, the currently opened folder is automatically selected as the "Save sent message in" folder. This should of course be a new option, because it departs from current behavior.

**Comment by alecpl** [comment]
We have Preferences > Composing Messages > Place replies in the folder of the message being replied to. Does that not solve your use-case? I think it makes sense for replies only, when I compose a new message I want it usually in Sent no matter which folder is open at a time.

**Comment by emtiu** [comment]
You're right: I use the "Place reples in …" feature, and it is useful to me. However, additionally, I sort outgoing (newly composed) in folders as well. In effect, I don't keep a Sent folder, which is why "Save newly composed mail in current folder" would be useful to me.

**Comment by kiceki** [comment]
My request have been closed for duplicate.
I second this report : The option would be way more usefull if it could also work with new messages when a folder is opened.
At least having a second option to unlock this specific feature for interested users.
Thanks.
